tests/lint/unittest_lint.py

```diff
@@ -706,38 +706,6 @@ def test_pylintrc_parentdir_no_package() -> None:
                     assert config.find_pylintrc() == expected
 
 
```

Comment:
I'm waiting on test coverage and will tests if needed, but these tests don't really work.

We need a `Run` object for `preprocess_options` now, which basically means all other integrated tests where the preprocessed arguments are tested directly do exactly the same (ie., set up a `Run` and parse args).
Felt unnecessary to keep this in, but will add tests if necessary to keep coverage.

Reply:
Added two tests that should cover the initial coverage drop :)